lib/ansible/modules/cloud/softlayer/sl_vm.py

```diff
@@ -170,8 +169,7 @@
   gather_facts: False
   tasks:
   - name: Build instances request
-    local_action:
```

Comment:
That's remote, running against localhost (which makes the action to be performed... on localhost)